### PR TITLE
Make aimpoint_mon into an installable package

### PR DESCRIPTION
**Pull request by taldcroft**
# Description

Make the cron utility files of this repo be installable as a Python package. There is a bunch of other stuff which is useful otherwise that is not packaged.

# Testing

- [N/A] Unit tests do not exist
- [x] Functional test

Installed this package into a local (on HEAD) Ska3-flight environment. Then:
```
cd /export/tom
bash Miniconda3-py38_4.8.3-Linux-x86_64.sh 
eval "$(/export/tom/miniconda3/bin/conda shell.bash hook)" 
conda create -n ska3-dev ska3-flight ska3-perl \
  --override-channels -c https://icxc.cfa.harvard.edu/aspect/ska3-conda/flight
conda activate ska3-dev

cd ~/git/aimpoint_mon  # package branch
python setup.py install --single-version-externally-managed --record=record.txt

ls $CONDA_PREFIX/share/aimpoint_mon
task_schedule.cfg

mkdir -p ~/tmp/data/aimpoint_mon
cd ~/tmp/data/aimpoint_mon/
rsync $SKA/data/aimpoint_mon/ ./
rm index*  # Remove existing output and templates

# Run the executable scripts that get run in the task_schedule.cfg cron job
aimpoint_mon_update_aimpoint_data --data-root=.
aimpoint_mon_update_observed_aimpoints --data-root=.
aimpoint_mon_plot_aimpoint --data-root=.
aimpoint_mon_make_web_page --data-root=.

# Examine index.html in browser ==> Looks OK

# Expected files updated
ls -lrt
...
-rw-rw-r-- 1 aldcroft aspect    6303880 Jan 29 16:20 aimpoint_asol_values.h5
-rw-rw-r-- 1 aldcroft aldcroft     2304 Jan 29 16:20 aimpoint_obsid_index.pkl
-rw-rw-r-- 1 aldcroft aldcroft 13746240 Jan 29 16:20 aimpoint_asol_values.fits
-rw-r--r-- 1 aldcroft aldcroft    72961 Jan 29 16:21 observed_aimpoints_dx.png
-rw-r--r-- 1 aldcroft aldcroft    68056 Jan 29 16:21 observed_aimpoints_dy.png
-rw-r--r-- 1 aldcroft aldcroft  2233658 Jan 29 16:21 observed_aimpoints_table.html
-rw-rw-r-- 1 aldcroft aspect     163319 Jan 29 16:21 chip_x_y_aciss.png
-rw-rw-r-- 1 aldcroft aspect     167344 Jan 29 16:21 chip_x_y_acisi.png
-rw-rw-r-- 1 aldcroft aspect      56164 Jan 29 16:21 intra_obs_dy_dz.png
-rw-rw-r-- 1 aldcroft aspect      29081 Jan 29 16:21 aca_housing_temperature.png
-rw-rw-r-- 1 aldcroft aspect       1893 Jan 29 16:21 info.json
-rw-rw-r-- 1 aldcroft aldcroft     5034 Jan 29 16:21 index.html

```

### CODE REVIEW

**Review by jeanconn — APPROVED** [review]
Looks good to me.  The few small-but-real code changes are harder to see in this packaging PR, but seem fine.  There are some outstanding uses of os.path type things that could have gone to pathlib, but I'm not passionate about them.

File: aimpoint_mon/make_web_page.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import argparse
+import json
+from pathlib import Path
+
+from jinja2 import Template
+import pyyaks.logger
+
+
+def get_opt():
+    parser = argparse.ArgumentParser(description='Get aimpoint drift data '
```
Comment (jeanconn) [inline aimpoint_mon/make_web_page.py:13]: I don't think that description was great before but not a new issue.

File: make_web_page.py [hunk 2]
```diff
@@ -1,49 +0,0 @@
...
-index_file = os.path.join(opt.data_root, 'index.html')
-info_file = os.path.join(opt.data_root, 'info.json')
-
-# Jinja template context
-logger.info('Loading info file {}'.format(info_file))
-context = json.load(open(info_file, 'r'))
-
-# Get the last record of asol aimpoint values
-h5 = tables.open_file(asol_file)
-last = h5.root.data[-1]
```
Comment (jeanconn) [inline make_web_page.py:40]: I think this was decorative before and you cut it when you reorganizes, which seems fine.

**Comment by taldcroft** [comment]
Let's leave actually using this as a project for 2021.3.

**Pull request merged**